src/System.Security.Cryptography.Primitives/src/System/Security/Cryptography/AsymmetricAlgorithm.cs

```diff
@@ -9,6 +9,9 @@ namespace System.Security.Cryptography
 {
     public abstract class AsymmetricAlgorithm : IDisposable
     {
+        protected int KeySizeValue;
+        protected KeySizes[] LegalKeySizesValue;
```

Comment:
Ok.  In that case, consider adding Assert.Equal/Nulls for these in the DoesNotSetLegalKeySizesField ctor.